scipy/special/tests/test_gammainc.py

```diff
@@ -53,6 +55,13 @@ def test_x_zero(self):
         a = np.arange(1, 10)
         assert_array_equal(sc.gammainc(a, 0), 0)
 
+    def test_nan_at_a_and_x_0(self):
+        assert np.isnan(sc.gammainc(0,0))
+
+    def test_limit_check(self):
+        result = sc.gammainc(1e-10, 1)
```

Comment:
I would assert that `sc.gammainc(1e-10, 1)` is close to `sc.gammainc(0, 1)`.

Reply:
done.